docker-installer/install_cni_plugins.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -e
+
+if [ $# -ne 2 ]; then
+  echo "USAGE: $0 source-dir dest-dir"
+  exit 1
+fi
+
+SRC=$1
+DST=$2
+
+install_cni_plugin() {
+	NAME=$1
+	MD5=$( md5sum $SRC/$NAME | awk '{ print $1 }' )
+	if [ -e $DST/$NAME ]; then
+		if [ ! -e $DST/$NAME.by_cni_installer_image ]; then
+			# The file already exists but there's no marker that
+			# it was installed by this installer -> keep untouched
+			echo "* '$NAME' ignored (exists but not installed by me)"
```

Comment:
So `FORCE=1` might be better to be here instead?

Reply:
I think it would be useful in both cases.

Reply:
Alright, so I'll add `FORCE` to force install all binaries in both cases.

Reply:
done.